src/main/java/org/wso2/lsp4intellij/client/languageserver/wrapper/MessageHandler.java

```diff
@@ -0,0 +1,43 @@
+package org.wso2.lsp4intellij.client.languageserver.wrapper;
+
+import java.util.function.Function;
+import org.eclipse.lsp4j.InitializeResult;
+import org.eclipse.lsp4j.jsonrpc.MessageConsumer;
+import org.eclipse.lsp4j.jsonrpc.messages.Message;
+import org.eclipse.lsp4j.jsonrpc.messages.ResponseMessage;
+import org.eclipse.lsp4j.services.LanguageServer;
+import org.jetbrains.annotations.NotNull;
+import org.wso2.lsp4intellij.client.languageserver.serverdefinition.ServerListener;
+
+class MessageHandler implements Function<MessageConsumer, MessageConsumer> {
```

Comment:
This class is package-protected. Is it intentional?

Reply:
@rasika Yes since its scope is only for the wrapper package ? Will it have any value by making it as public ?